### PR TITLE
refactor: migrate score data from db to .pt file

### PR DESCRIPTION
**Pull request by codebender37**

### CODE REVIEW

File: .env.miner.example [hunk 1]
```diff
@@ -48,3 +48,6 @@ S3_PUBLIC_URL=
 JWT_SECRET=
 # e.g. infura, alchemy API url for ethereum
 ETHEREUM_NODE=
+
```
Comment (jarvis8x7b) [inline .env.miner.example]: can we shift this to the top? this section is meant for decentralized env vars

File: dojo/utils/uids.py [hunk 2]
```diff
@@ -29,6 +29,7 @@ def is_miner(metagraph: bt.metagraph, uid: int) -> bool:
     stakes = metagraph.S.tolist()
     from dojo import VALIDATOR_MIN_STAKE
 
+    print(f"Validator min stake {VALIDATOR_MIN_STAKE}")
```
Comment (jarvis8x7b) [inline dojo/utils/uids.py]: use bt logging

**Review by jarvis8x7b — CHANGES_REQUESTED** [review]

File: dojo/utils/uids.py (hunk 2 above)
Comment (codebender37) [reply dojo/utils/uids.py]: oh, sorry forget to remove

**Review by karootplx — APPROVED** [review]
lgtm

File: .env.miner.example [hunk 3]
```diff
@@ -17,6 +17,7 @@ SUBTENSOR_ENDPOINT=wss://entrypoint-finney.opentensor.ai:443
 # NETUID=98
 # SUBTENSOR_NETWORK=test
 # SUBTENSOR_ENDPOINT=ws://testnet-lite:9944
+# VALIDATOR_MIN_STAKE=1
```
Comment (jarvis8x7b) [inline .env.miner.example]: set this to 20k so that if the miner forgets it's set properly

**Review by jarvis8x7b — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -278,6 +278,7 @@ DOJO_API_KEY= # blank for now
 WALLET_COLDKEY=# the name of the coldkey
 WALLET_HOTKEY=# the name of the hotkey
 AXON_PORT=8888 # port to serve requests over the public network for validators to call
+VALIDATOR_MIN_STAKE= # minimum stake required for validators default is 20000 TAO (use this to bypass the blacklist function in testnet)
```
Comment (jarvis8x7b) [inline README.md]: can set to the default 20k value

**Review by jarvis8x7b — APPROVED** [review]

**Pull request merged**